src/ai/backend/runner/entrypoint.sh

```diff
@@ -102,7 +102,13 @@ else
 
   # Start ssh-agent if it is available
   if command -v ssh-agent > /dev/null; then
-    eval "$(ssh-agent -s)"
+    SSH_ENV_PATH="/home/work/ssh-agent.env"
+
+    /opt/kernel/su-exec $USER_ID:$GROUP_ID ssh-agent > $SSH_ENV_PATH
+    chmod +x $SSH_ENV_PATH
+    . $SSH_ENV_PATH
+    rm $SSH_ENV_PATH
+
```

Comment:
How about reducing this to:
```suggestion
    eval "$(/opt/kernel/su-exec $USER_ID:$GROUP_ID ssh-agent)"
```

Using `eval "$(...)"` is orthogonal to using `su-exec`.